WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportFormViewModel.swift

```diff
@@ -156,6 +185,21 @@ extension SupportFormViewModel {
     }
 }
 
+// MARK: Private helpers
+private extension SupportFormViewModel {
+    func handleZendeskIdentityIfNeeded() {
```

Comment:
nit: wdyt about calling it `requestZendeskIdentityIfNeeded()` ? That will imply that the app needs to ask users for some input (while `handling` sounds like the app will take care of it automatically)

Not a big deal, though.

Reply:
Updated in ef1ea201094272b4120335de3636e2f01b0b26c3.